src/config/__jest__/Config.test.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import Config from '../Config';
+import lodash from 'lodash';
+
+jest.mock('fs');
+describe('Default variables', () => {
+  let testedConfig = new Config();
+  test('Default clientID and token are empty', () => {
+    expect(testedConfig.clientId).toEqual('');
+    expect(testedConfig.token).toEqual('');
+  });
+  test('Default language is English', () => {
+    expect(testedConfig.language).toEqual('en');
+  });
+  test('Default command prefix is !', () => {
+    expect(testedConfig.prefix).toEqual('!');
+  });
+  test('Default whitelisted extensions are mp3 and wav files', () => {
+    expect(testedConfig.acceptedExtensions).toEqual(['.mp3', '.wav']);
+  });
+  test('Default maximum filesize is ', () => {
+    expect(testedConfig.maximumFileSize).toEqual(1000000);
+  });
+  test('Default setting to delete messages is false', () => {
+    expect(testedConfig.deleteMessages).toBeFalsy();
```

Comment:
Let's not use `toBeFalsy` / `toBeTruthy`. The issue is the following

`0`, `""`, `null`, `undefined`, `NaN` are `falsy`, **everything else** is `truthy`. So the test would pass even if the value is not specifically what we want. Let's use `toBe(true)` / `toBe(false)` instead.